subcommand/sync-catalog/command.go

```diff
@@ -123,6 +124,9 @@ func (c *Command) init() {
 		"namespace mirroring")
 	c.flags.StringVar(&c.flagK8SNSMirroringPrefix, "k8s-namespace-mirroring-prefix", "",
 		"[Enterprise Only] Prefix that will be added to all k8s namespaces mirrored into Consul if mirroring is enabled.")
+	c.flags.StringVar(&c.flagCrossNamespaceACLPolicy, "cross-consul-namespace-acl-policy", "",
```

Comment:
Would it make more sense to name it `consul-cross-namespace-acl-policy` to match other names?

Reply:
I was trying to specify that it was for crossing consul namespaces, but it does end up being a little awkward. I'll update it.